providers/openfeature-provider-flagd/README.md

```diff
@@ -36,12 +49,17 @@ api.set_provider(FlagdProvider(
 
 The default options can be defined in the FlagdProvider constructor.
 
-| Option name    | Type & Values | Default   |
-|----------------|---------------|-----------|
-| host           | str           | localhost |
-| port           | int           | 8013      |
-| schema         | str           | http      |
-| timeout        | int           | 2         |
+| Option name                   | Type & Values | Default   |
```

Comment:
Can you set these properties via environment variables?

Reply:
Good point, could be a follow-up to add env car support matching other providers (see java [1])

[1] - https://github.com/open-feature/java-sdk-contrib/tree/main/providers/flagd#configuration-options

Reply:
Yes, for all except timeout for some reason. Let me know if you'd like to tweak naming for example changing `timeout` to `deadline`. I added the env vars to readme for visibility